main.go

```diff
@@ -1,19 +1,57 @@
 package main
 
 import (
+	"errors"
```

Comment:
Could we make it "github.com/pkg/errors" too.

Reply:
Sure.